common/KeeNotification.ts

```diff
@@ -43,6 +44,22 @@ class KeeNotification {
         return container;
     }
 
+    renderCloseButton (container: HTMLDivElement) {
+        const button = document.createElement("span");
+        
+        button.classList.add("close-button", "glyphicon", "glyphicon-remove");
+        // TODO: should we add hint to this obvious action button? 
+        // in this case we need i18n for this hint
+        // button.setAttribute("title", "Hide notification");
+
+        button.addEventListener("click", function () {
+            this.myPort.postMessage({ removeNotification: this.id } as AddonMessage);
+        }.bind(this));
```

Comment:
I tend to prefer arrow functions rather than binding this so please can you change this to:

````
button.addEventListener("click", () => {
            this.myPort.postMessage({ removeNotification: this.id } as AddonMessage);
        });
````